.ci/run.sh

```diff
@@ -30,24 +30,27 @@ sudo docker exec ${container_id} echo hello
 sudo docker ps -a
 
 # try to unpause a container that isn't paused a few times
+tmpfile=$(mktemp)
 for i in 1 2 3
 do
-	{ sudo docker unpause ${container_id}; ret=$?; } || true
-	[ $ret -eq 1 ]
+	{ sudo docker unpause ${container_id} >"$tmpfile" 2>&1; ret=$?; } || true
+	[ $ret -eq 1 ] || { cat "$tmpfile"; exit 1; }
```

Comment:
Q. does docker always returns ```1``` on failure? wouldn't be better validate against ```0``` ?